projects/manager/.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+    "extends": "../../.eslintrc.json",
+    "ignorePatterns": ["!**/*"],
+    "overrides": [
+        {
+            "files": ["*.ts"],
+            "parserOptions": {
+                "project": ["projects/manager/tsconfig.app.json", "projects/manager/tsconfig.spec.json"],
+                "createDefaultProgram": true
+            },
+            "rules": {}
+        },
+        {
+            "files": ["*.html"],
+            "rules": {}
+        }
+    ]
+}
```

Comment:
I guess datasets should in particular display (and later have it editable) their loading information.

Reply:
yes! I guess this is one of the most challenging tasks of the manager. Because we need at least one component for every MetaData type. I don't think I can complete this in the scope of this task though.

Reply:
you can show the JSON for now?

Reply:
8a259f263f4fdad8909d38cee8ac02d4484f10f2